content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,556 @@
+  ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+  ```shell 
+  npm install sharetribe-flex-sdk
+  
+  npm install sharetribe-flex-integration-sdk
+  ```
+  </Tabs.Tab>
+</Tabs>
+{/* // prettier-ignore */}
+
+## SDK Initialization Overview
+
+### Obtaining the Client ID and secret
+
+To use the Marketplace API or Integration API, you will need a **client
+ID** from an application. You can create an application in Sharetribe
+Console: _Build > Advanced > Applications_
+
+![Sharetribe Console: Applications tab](./console-build-application.png)
+
+You should also make note of the **client secret**, as some requests
+require a secure context and this can only be done when initializing the
+SDK with both the client ID and secret.
+
+An example of this type of request is making a privileged transition in
+a transaction. This requires that the:
+
+- Client ID and secret are used when creating an instance of the SDK
+- A token exchange happens to obtain a trusted user access token
+- The request is made from a secure context.
+
+<Callout type="info">
+  In the [Sharetribe Web
+  Template](https://github.com/sharetribe/web-template), these types of
+  requests are made on a small Node server included in the Template.
+</Callout>
+
+### Initializing a Sharetribe SDK instance
+
+You can initialize a Sharetribe SDK instance using only the
+application's Client ID. Ensure the client ID is from a **Marketplace
+API application** created in Console.
+
+```jsx
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+const sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+});
+```
+
+This will allow you to make requests that don't require authentication
+or a trusted SDK instance, e.g. querying listings.
+
+### Initializing an Integration SDK instance
+
+If you are accessing Integration API, you'll need to also provide the
+client secret. Ensure the client ID is from an **Integration API
+application** created in Console.
+
+```jsx
+const sharetribeIntegrationSdk = require('sharetribe-flex-integration-sdk');
+
+const integrationSdk = sharetribeIntegrationSdk.createInstance({
+  clientId: '<Client ID>',
+  clientSecret: '<Client secret>',
+});
+```
+
+<Callout type="warning">
+  Keep your application's client secret secure. Never expose it to
+  untrusted devices or applications such as end user browsers or mobile
+  apps.
+</Callout>
+
+### Initializing the SDK with a trusted user access token
+
+To make certain requests, you will need to access the Marketplace API
+from a trusted context (i.e. on the server). In order to do this, you'll
+need to log in using the SDK and perform a token exchange to retrieve a
+trusted token.
+
+<Steps>
+
+{<h3>Create an instance of the SDK with the client ID, secret, and token
+store</h3>}
+
+```jsx
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+var sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+  clientSecret: '<Client secret>',
+  tokenStore: sharetribeSdk.tokenStore.memoryStore(),
```

Comment:
I would remove this.

This depends a bit about the environment in which this piece of code is run. But since it's Marketplace API, we can assume that the user login happens in the marketplace frontend. The default store is cookie store, which allows the frontend and backend share the same access token.